Decred Wallet/view_controller/ReceiveViewController.swift

```diff
@@ -36,8 +48,15 @@ class ReceiveViewController: UIViewController,UIDocumentInteractionControllerDel
                self.showFirstWalletAddressAndQRCode()
```

Comment:
No need to call this function if the generated address/QR code will not be displayed.